### PR TITLE
Implement `has_focus` for NativeComboBox

### PR DESCRIPTION
**Pull request by notfirefox**
**Changes**
- Implement `has_focus` for NativeComboBox

### CODE REVIEW

**Comment by notfirefox** [comment]
@ogoffart Should be good to go. The `node_test` fails though.

**Comment by ogoffart** [comment]
> The node_test fails though.

Yes, that test has been flaky for some time :-(

**Review by ogoffart — APPROVED** [review]
thanks!

File: internal/compiler/builtins.slint [hunk 1]
```diff
@@ -501,6 +501,7 @@ export component NativeTableHeaderSection {
 export component NativeComboBox {
     in-out property <string> current_value;
     in property <bool> enabled: true;
+    in property <bool> has-focus;
```
Comment (ogoffart) [inline internal/compiler/builtins.slint:504]: Apparently it needs to be `out`  (as the test says)
But then you might not be able to set it from the ComboBox.
I need to think how best to solve this.

**Comment by ogoffart** [comment]
Moved to #5061

**Pull request closed without merging**